src/Trakx.YouSign.ApiClient/Trakx.YouSign.ApiClient.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <GenerateApiClient>False</GenerateApiClient>
+    <AssemblyName>Trakx.YouSign.ApiClient</AssemblyName>
+    <RootNamespace>Trakx.YouSign.ApiClient</RootNamespace>
+    <RepositoryUrl>https://github.com/trakx/yousign-api-client</RepositoryUrl>
```

Comment:
So maybe this should come from the packable or the Common.Projects.props, then I am not sure it is worth bothering about the namespace and rootnamespace explicitly, but I might be wrong.
It is just easier to refactor if these are not replicated in many places in the code ;)

Reply:
Unwated settings were removed.